python/deltalake/table.py

```diff
@@ -638,3 +615,87 @@ def get_add_actions(self, flatten: bool = False) -> pyarrow.RecordBatch:
         2  x=1/0-91820cbf-f698-45fb-886d-5d5f5669530b-0.p...         565 1970-01-20 08:40:08.071         True            1            1             0      4      4
         """
         return self._table.get_add_actions(flatten)
+
+
+class TableOptimizer:
+    """API for various table optimization commands."""
+
+    def __init__(self, table: DeltaTable):
+        self.table = table
+
+    def __call__(
+        self,
+        partition_filters: Optional[List[Tuple[str, str, Any]]] = None,
```

Comment:
We use this type in various places, what do you think about aliasing this to maybe `PartitionFilters`?

Reply:
Sounds good. We had already written the type in a previous PR.